guides/rules/no-jquery-methods.md

```diff
@@ -0,0 +1,6 @@
+# No jQuery Methods
+
+**TL;DR** The use of the specific blacklist jQuery method has been deprecated.
```

Comment:
"The intent of this rule is to identify any blacklisted jQuery methods as deprecated"